internal/bzlmod/go_deps.bzl

```diff
@@ -577,7 +694,9 @@ _config_tag = tag_class(
 
 _from_file_tag = tag_class(
     attrs = {
-        "go_mod": attr.label(mandatory = True),
+        "go_mod": attr.label(mandatory = False),
+        "go_work": attr.label(mandatory = False),
+        "fail_on_version_conflict": attr.bool(default = True),
```

Comment:
Let's replace this with a constant `True` for now as it gives us more flexibility to experiment with tolerating this situation in the future (e.g. MVS).

Reply:
Internally we have relied on this being configurable. This has been quite helpful to our engineerings when dealing with larger migrations to the new form. Would it be OK to continue to support that use-case?